lib/setup/setupVisDebug.js

```diff
@@ -91,13 +91,13 @@ function VisDebug(options) {
         objects.writeFile('vis', 'edit.html', file);
 
         console.log('Modify "' + path.normalize(visDir + '/www/cache.manifest') + '"');
-        file = fs.readFileSync(visDir + '/www/cache.manifest').toString();
+        file = fs.readFileSync(visDir + '/www/cache.manifest', 'utf8');
         var n = file.match(/# dev build (\d+)/, '5');

```

Comment:
VSCode complains that `match` only takes one argument. What does the '5' do?

Reply:
Good question, I would say we remove it?!